### PR TITLE
Router

### LINKED ISSUE #18: figure out how Router works

### PR DESCRIPTION
**Pull request by Salamandastron1**
____ Wrote Tests
____ Implemented 
____ Reviewed
# Neccesary checkmarks:
- [] All Tests are Passing

- [x] The code will run locally

## Type of change
- [x] New feature 
- [] Bug Fix

# Implements/Fixes:
* description
closes #18 

# Check the correct boxes

- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

# Please Include a link to a gif of your feelings about this branch
Link:  https://media.giphy.com/media/Wy3yu6suaOEgM/giphy.gif

### CODE REVIEW

**Comment by Salamandastron1** [comment]
@Jessica-Erickson if you could review this that would be most helpful

File: .eslintrc.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
       "function-paren-newline": 0,
       "func-names": 0,
       "implicit-arrow-linebreak": 0,
-      "no-use-before-define": ["error", { "variables": false }]
+      "no-use-before-define": ["error", { "variables": false }],
+      "react/prop-types": ["error", { "ignore": ["navigation"] }]
```
Comment (SoleraOps) [inline .eslintrc.json:11]: Why does navigation not need proptypes?

File: components/App.js [hunk 2]
```diff
@@ -13,18 +26,7 @@ export default class App extends React.Component {
...
     return (
-      <View style={styles.container}>
-        <LandingPage />
-      </View>
+      <AppContainer />
     );
   }
 }
-
-const styles = StyleSheet.create({
```
Comment (SoleraOps) [inline components/App.js:23]: Where do styles live now?

**Review by SoleraOps — APPROVED** [review]
Looks great!

File: .eslintrc.json (hunk 1 above)
Comment (Salamandastron1) [reply .eslintrc.json:11]: There was an issue where bringing in the 'react-native' function ```navigation.navigate``` meant that I could not figure out how to test both parts of the prop. It ended up taking too long to figure out so I settled on this solution for the linter.

**Pull request merged**